### DISCUSSION TITLE
query only work with some collections but did not work with other

### DISCUSSION THREAD

**Discussion by TyLy83** [invertase/react-native-firebase#5552] [General]
query only work with some collections but did not work with other? please help

**Comment by mikehardy** [comment]
With apologies, this is not nearly enough information to go on. Typically a bug report should include something like an App.js we can use to probe the issue (https://stackoverflow.com/help/minimal-reproducible-example) if we drop it in https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh - and usage questions should go on stackoverflow.com where there is a good community of firebase users (from more than just the react-native-firebase area) and you have a good chance of receiving help
